lib/rspec-puppet/example.rb

```diff
@@ -8,14 +10,15 @@
 require 'rspec-puppet/example/provider_example_group'
 require 'rspec-puppet/example/application_example_group'
 
-RSpec::configure do |c|
-
+RSpec.configure do |c|
   def c.rspec_puppet_include(group, type, file_path)
     escaped_file_path = Regexp.compile(file_path.join('[\\\/]'))
     if RSpec::Version::STRING < '3'
```

Comment:
I think you can also drop support for RSpec 2 instead of changing the code below.